src/app/admin/volunteers/shared/newvolunteer-register-form/newvolunteer-register-form.component.scss

```diff
@@ -0,0 +1,160 @@
+$gapBetweenFieldAndHint: "25px";
+$fontFamily: Roboto, sans-serif;
```

Comment:
font-family is set globaly

Reply:
$fontFamily: Roboto, sans-serif; - removed as requested.